src/views/checkups/utils/utils.ts

```diff
@@ -14,22 +14,25 @@ export const STATUS_TIMEOUT = 'spec.timeout';
 export const STATUS_START_TIME_STAMP = 'status.startTimestamp';
 export const STATUS_FAILURE_REASON = 'status.failureReason';
 export const STATUS_SUCCEEDED = 'status.succeeded';
-export const STATUS_COMPILATION_TIME_STAMP = 'status.completionTimestamp';
+export const STATUS_COMPLETION_TIME_STAMP = 'status.completionTimestamp';
 export const CONFIGMAP_NAME = 'CONFIGMAP_NAME';
+export const CONFIGMAP_NAMESPACE = 'CONFIGMAP_NAMESPACE';
 
 export const generateWithNumbers = (name: string): string =>
   `${name}-${Math.floor(Math.random() * 10000)}`;
 
-export const findObjectByName = <T extends K8sResourceCommon>(arr: T[], name: string): T =>
-  (arr || []).find((obj) => obj?.metadata?.name === name);
+export const findObjectByName = <T extends K8sResourceCommon>(
+  arr: T[],
+  name: string,
+): T | undefined => (arr || []).find((obj) => obj?.metadata?.name === name);
 
 export const columnsSorting = (
   data: IoK8sApiCoreV1ConfigMap[],
   sortDirection: SortByDirection,
   field: string,
   alternativeField = '',
 ) =>
-  data.sort((a, b) => {
+  data.toSorted((a, b) => {
```

Comment:
`toSorted()` failed for me inside unit tests - it seems we are running them on older Node. Not sure if there are other blockers - to be checked/discussed.